mobile/src/reducers/people/people.reducer.ts

```diff
@@ -0,0 +1,36 @@
+import { Reducer } from 'redux';
+import { Map, Set } from 'immutable';
+import { combineEpics } from 'redux-observable';
+
+import { User } from '../user/user.model';
+import { Employee } from '../organization/employee.model';
+import { OrganizationActions } from '../organization/organization.action';
+import { UserActions } from '../user/user.action';
+import { PeopleActions } from './people.action';
+import { loadUserDepartmentEmployeesEpic$ } from './people.epics';
+import { EmployeeMap } from '../organization/employees.reducer';
+
+export interface PeopleState {
+    employees: EmployeeMap;
+}
+
+const initState: PeopleState = {
+    employees: Map()
+};
+
+export const peopleEpics = combineEpics(
+    loadUserDepartmentEmployeesEpic$ as any);
+
+export const peopleReducer: Reducer<PeopleState> = (state = initState, action: PeopleActions) => {
```

Comment:
we have to discuss it tomorrow, I'm not sure what's the point of the separate reducer. Did @serjKim say to do it that way? :)

Reply:
yes, initially